src/libraries/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -2076,6 +2077,95 @@ public async Task SendHttpGetRequest_WithCachePolicyBypassCache_DoNotAddHeaders(
             });
         }
 
+        [Fact]
+        public async Task SendHttpPostRequest_WithContinueTimeoutAndBody_BodyIsDelayed()
+        {
+            await LoopbackServer.CreateClientAndServerAsync(
+                async (uri) =>
+                {
+                    HttpWebRequest request = WebRequest.CreateHttp(uri);
+                    request.Method = "POST";
+                    request.ServicePoint.Expect100Continue = true;
+                    request.ContinueTimeout = 30000;
+                    Stream requestStream = await request.GetRequestStreamAsync();
+                    requestStream.Write("aaaa\r\n\r\n"u8);
+                    var _ = request.GetResponseAsync();
```

Comment:
Can we avoid the fire-and-forget task here, something like `await Assert.ThrowsAnyAsync<Exception>() => request.GetResponseAsync());`?
(same for other tests below)

Reply:
It would be nice to clean all Tasks before test is done. Either as @MihaZupan suggested, store and await the Task later to using `.ContinueWith` as we do in some places in HTTP.